src/components/menu/menu-test.stories.tsx

```diff
@@ -210,3 +213,489 @@ InNavigationBarStory.story = {
+            </MenuItem>
+            <MenuItem submenu="Menu Item Four" onClick={() => {}}>
+              <MenuItem onClick={() => {}}>Item Submenu One</MenuItem>
+              <MenuSegmentTitle>segment title</MenuSegmentTitle>
+              <MenuItem href="#">Item Submenu Two</MenuItem>
+            </MenuItem>
+          </Menu>
+        </div>
+      ))}
+    </Box>
+  );
+};
+
+export const MenuComponentScrollable = ({ ...props }) => {
+  const menuTypes = ["white", "light", "dark", "black"] as const;
+  return (
+    <Box mb={150}>
+      {menuTypes.map((menuType) => (
+        <div key={menuType}>
+          <Typography variant="h4" textTransform="capitalize" my={2}>
+            {menuType}
+          </Typography>
+          <Menu menuType={menuType}>
+            <MenuItem onClick={() => {}}>Menu Item One</MenuItem>
+            <MenuItem href="#">Menu Item Two</MenuItem>
+            <MenuItem submenu="Menu Item Three">
+              <ScrollableBlock height="200px" {...props}>
+                <MenuItem href="#">Item Submenu One</MenuItem>
+                <MenuItem href="#">Item Submenu Two</MenuItem>
+                <MenuItem href="#">Item Submenu Three</MenuItem>
+                <MenuItem href="#">Item Submenu Four</MenuItem>
+                <MenuItem href="#">Item Submenu Five</MenuItem>
+                <MenuItem href="#">Item Submenu Six</MenuItem>
+                <MenuItem href="#">Item Submenu Seven</MenuItem>
+                <MenuItem href="#">Item Submenu Eight</MenuItem>
+                <MenuItem href="#">Item Submenu Nine</MenuItem>
+                <MenuItem href="#">Item Submenu Ten</MenuItem>
+                <MenuItem href="#">Item Submenu Eleven</MenuItem>
+                <MenuItem href="#">Item Submenu Twelve</MenuItem>
+              </ScrollableBlock>
+            </MenuItem>
+            <MenuItem submenu="Menu Item Four">
+              <MenuItem href="#">Item Submenu One</MenuItem>
+              <MenuItem href="#">Item Submenu Two</MenuItem>
+              <ScrollableBlock variant="alternate" height="200px">
+                <MenuItem href="#">Item Submenu Three</MenuItem>
+                <MenuItem href="#">Item Submenu Four</MenuItem>
+                <MenuItem href="#">Item Submenu Five</MenuItem>
+                <MenuItem href="#">Item Submenu Six</MenuItem>
+                <MenuItem href="#">Item Submenu Seven</MenuItem>
+                <MenuItem href="#">Item Submenu Eight</MenuItem>
+                <MenuItem href="#">Item Submenu Nine</MenuItem>
+                <MenuItem href="#">Item Submenu Ten</MenuItem>
+                <MenuItem href="#">Item Submenu Eleven</MenuItem>
+                <MenuItem href="#">Item Submenu Twelve</MenuItem>
+              </ScrollableBlock>
+            </MenuItem>
+          </Menu>
+        </div>
+      ))}
+    </Box>
+  );
+};
+
+export const MenuComponentSearch = () => {
+  const menuTypes = ["white", "light", "dark", "black"] as const;
```

Comment:
we can specify this const above, because we are using it in almost all components